src/reports/components/formatted-date.tsx

```diff
@@ -1,61 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { padStart } from 'lodash';
+import * as moment from 'moment';
 import * as React from 'react';
 
 export interface FormattedDateProps {
     date: Date;
 }
 
-type timeRange = 'AM' | 'PM';
-
 export class FormattedDate extends React.Component<FormattedDateProps> {
     public render(): JSX.Element {
         return <>{this.formatDateTime(this.props.date)}</>;
     }
 
     private formatDateTime(date: Date): string {
-        return (
-            date.getFullYear() +
-            '-' +
-            this.padStartWithZero(date.getMonth() + 1, 2) +
-            '-' +
-            this.padStartWithZero(date.getDate(), 2) +
-            ' ' +
-            this.getTwelveHour(date.getHours()) +
-            ':' +
-            this.padStartWithZero(date.getMinutes(), 2) +
-            ' ' +
-            this.getAmPm(date.getHours()) +
-            ' ' +
-            this.getTimeZone(date)
-        );
-    }
-
-    private getTwelveHour(hour: number): number {
-        hour = hour % 12;
-        if (hour === 0) {
-            return 12;
-        }
-        return hour;
-    }
-
-    private getAmPm(hour: number): timeRange {
-        if (hour < 12) {
-            return 'AM';
-        } else {
-            return 'PM';
-        }
-    }
-
-    private getTimeZone(date: Date): string {
-        const timeString = date
-            .toLocaleTimeString('en-us', { timeZoneName: 'short' })
-            .replace(/\u200E/g, '');
-        return timeString.substr(timeString.lastIndexOf(' ') + 1);
-    }
-
-    private padStartWithZero(num: number, digits: number): string {
-        return padStart(num.toString(), digits, '0');
+        let momentDate = moment(this.props.date);
+        return momentDate.toLocaleString();
```

Comment:
I'm pretty sure this isn't right. Moment doesn't document a `toLocalString()` implementation at all, so I think this is going to actually be picking up `Object.prototype.toLocaleString()`, which falls back to the moment object's plain `toString()` since it doesn't implement a `toLocaleString()`.

What you actually would want is to call `format`, passing it one of the "Localized format" options [it documents](https://momentjs.com/docs/#/displaying/format/).

Note that the default `'moment'` package only contains an english locale, not all locales; to get support for other locales, you'd want to change the import line to be `from 'moment/min/moment-with-locales'` rather than just `from 'moment'`.

Even with the correct package imported, Moment infers the local timezone but it does *not* infer the local language information by default. You would need to tell it explicitly which locale to use. The correct way to infer that from modern versions of Firefox/Chromium is `window.navigator.language`, though we'd typically use a `windowUtils` around the direct navigator access for mockability.

Be:

```ts
import * as moment from 'moment/min/moment-with-locales';

const momentDate = moment(this.props.date);
const localMomentDate = momentDate.locale(this.props.deps.windowUtils.language() /* returns window.navigator.language */);
const formattedDate = localMomentDate.format('LTS');
```